src/transactions.rs

```diff
@@ -184,16 +200,20 @@ impl<'a, L: Ledger + Serialize + DeserializeOwned> TransactionEditor<'a, L> {
     }
 
     /// Add the UIDs of memos we are waiting on to complete the transaction
-    pub fn add_pending_uids(mut self, uids: &[u64]) -> Self {
+    pub fn add_pending_uids(mut self, uids: Vec<u64>) -> Self {
```

Comment:
Why this change? If we don't require ownership of the `Vec`, taking a slice is a bit better because it allows the function to be called in more situations without cloning.

Reply:
I will take another look at this.  I was having some trouble compiling this when the param was a slice.  But now I can't remember the exact issue.  Revert this change or post the issue if I hit it and can't solve it.